README.md

```diff
@@ -1,4 +1,107 @@
+- Transformer & Serializer now take MarkdownFile objects
+- Serializer can now handle media with relative file paths.
+- Added option: "anki.md.card.strategy" with "Nested Headers" & "Delimiter"; Delimiter is original author strategy. See Nested Headers below
+- Added option: "anki.send.allowUpdates". If enabled, this will check the anki db for cards where the front matches the sent card. Then, if the rest of the fields are not a match, it will delete the card from the anki db and replace it with the sent card. This will allow you continuously edit the markdown files and have it reflected in anki. This will of course, remove any learning state from the card, which is my current intended behavior. Beware: If you edit a card from within Anki, these changes will trigger a delete as it will no longer match the markdown.
+- Added option: "anki.send.keepSync". If enabled, after send metadata is added to the bottom of the file. Subsequent file changes will trigger another send. BEWARE: Cards deleted from the markdown will be deleted from the Anki database. Delete the metadata from the file to prevent future sync.
+- Fixed: pushNewCardsToAnki returns NOTE IDs, which were assigned to the Card model's id field. AnkiSerivce.findCards then assigned CARD IDS to the Id field. Fixed to assign note Ids instead of Card Ids in the Card model. Anki potentially creates multiple cards per note, so if there is a need to track card ids, i renamed Card.id to Card.noteId to prevent future confusion.
+- Send to Deck now saves file afterward if user has enabled Sync to allow writing metadata to the filesystem.
+- Send is now more informative, reporting counts of changes.
+- Send now resets Anki UI to DeckBrowser (important if card is deleted that user is actively reviewing)
+
+## Todo:
+
+- System to designate files/dirs for exclusion/inclusion on Send All
+- Add option so that card updates are in place, leaving the learning meta data.
+- Option to display a Markdown->Anki deck diff popup with option proceed or cancel. Might be useful if not sure which cards will be created/deleted.
+- Display hover-over link to show the corresponding Anki card in the guiBrowser. I imagine this similar to Yomichan little book icon, except appearing at the end of the header lines.
+- Need to check to make sure text above any header at the top of a document is handled appropriately.
+- Test: What happens if the user hit save multiple times rapidly before Send To Deck completes when it occurs on save?
+- Don't send if no changes to file. Important for Send All command so it doesn't check every file every time needlessly. Probably need to add last-sent meta timestamp to compare to file mod time.
+- Make sure added features are working with "Send to Own Deck"
+- Auto-hide/collapse anki metadata in editor so you don't have to look at it.
+
+## Known Issues
+
+- What should SendAll do if multiple workspaces? 
```

Comment:
Can we remove the sendAll feature from this? It sounds like there's already a lot to do with the Sync alone without overcomplicating it. 

We can add sendAll at a later date

Reply:
Yeah. I actually don't understand the GitHub pull request process. I can I suppose make a simpler branch on my end and send that instead, right?

Reply:
Yep!